testing/run_tests.py

```diff
@@ -480,11 +478,26 @@ def RunFrontEndServerTests(build_dir):
 def RunConstFinderTests(build_dir):
   test_dir = os.path.join(buildroot_dir, 'flutter', 'tools', 'const_finder', 'test')
   opts = [
+    '--disable-dart-dev',
     os.path.join(test_dir, 'const_finder_test.dart'),
     os.path.join(build_dir, 'gen', 'frontend_server.dart.snapshot'),
     os.path.join(build_dir, 'flutter_patched_sdk')]
   RunEngineExecutable(build_dir, os.path.join('dart-sdk', 'bin', 'dart'), None, flags=opts, cwd=test_dir)
 
+def RunLitetestTests(build_dir):
```

Comment:
Please add a smoke test to this that verifies that a failing test causes the whole harness to exit with a failure exit code.

Reply:
Done. Repurposed the already existing `testing/smoke_test_failure`.